src/components/FormField/commonElement.tsx

```diff
@@ -0,0 +1,111 @@
+import styled, { css } from "styled-components";
+
+export const FormElement = css`
+  display: flex;
+  align-items: center;
+
+  padding: 0.34375rem 0.75rem;
+  justify-content: space-between;
+  align-items: center;
+  border-radius: 0.25rem;
+  border: 1px solid var(--click-field-color-stroke-default, #e6e7e9);
```

Comment:
let's use theme instead of css variables as we will have autocomplete on theme. 
css variables might not be updated currently